### PR TITLE
Update docs.

### PR DESCRIPTION
**Pull request by aleksei-korolev**
GitHub screwed up the diff in the configuration.md, sorry for the inconvenience.

I tried to simplify the bindAddress parameter decsriptions, but I'm not sure if I got it right, I'm not sure what is listening to what and where.

### CODE REVIEW

**Comment by aleksei-korolev** [comment]
@iotaledger/tech-writers

File: documentation/docs/reference/api_reference.md [hunk 1]
```diff
@@ -16,6 +16,6 @@ keywords:
 
 # API Reference
 
-You can find specifications of the API exposed by INX-POI in the links below:
+You can find specifications of the INX-POI API by the link below:
```
Comment (lucas-tortora) [inline documentation/docs/reference/api_reference.md]:
```suggestion
You can find the specifications for the INX-POI API in the following link:
```

File: documentation/docs/reference/configuration.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+```bash
+inx-poi -h --full
+```
+
+## <a id="app"></a> 1. Application
+
+| Name            | Description                                                                                            | Type    | Default value |
+| --------------- | ------------------------------------------------------------------------------------------------------ | ------- | ------------- |
+| checkForUpdates | If true, the application will check for updates.                                                       | boolean | true          |
+| stopGracePeriod | The maximum time for background processes to finish before the app would terminate on system shutdown. | string  | "5m"          |
```
Comment (lucas-tortora) [inline documentation/docs/reference/configuration.md]:
```suggestion
| stopGracePeriod | The maximum time for background processes to finish before the app terminates on system shutdown. | string  | "5m"          |
```

File: documentation/docs/welcome.md [hunk 3]
```diff
@@ -18,25 +18,24 @@ keywords:
 
 # Welcome to INX-POI
 
-INX-POI enables you to generate and verify Proof-of-Inclusion of blocks in the Tangle.
-Given a piece of data or transaction and the proof, you can verify whether it was included in the Tangle at any given time.
+INX-POI generates and verifies proofs of inclusion of blocks. By generating a proof, you can always verify that the block used to be part of the Tangle. The Tangle nodes truncate old blocks by necessity, but with proofs of inclusion you can preserve important blocks on your own.
```
Comment (lucas-tortora) [inline documentation/docs/welcome.md]:
```suggestion
INX-POI generates and verifies proofs of inclusion of blocks. By generating a proof, you can always verify that the block used to be part of the Tangle. The Tangle nodes truncate old blocks by necessity, but with proofs of inclusion, you can preserve important blocks on your own.
```

File: documentation/docs/welcome.md [hunk 4]
```diff
@@ -18,25 +18,24 @@ keywords:
...
 
 ## Setup
```
Comment (lucas-tortora) [inline documentation/docs/welcome.md:23]:
```suggestion
## Set Up
```

File: documentation/docs/welcome.md [hunk 5]
```diff
@@ -18,25 +18,24 @@ keywords:
...
 ## Configuration
 
 The extension will connect to your local HORNET instance by default.
 
-You can find all the configuration options in the [configuration section](configuration.md).
+You can find all the configuration options in the [configuration section](reference/configuration.md).
 
 ## API
 
 The extension exposes a custom set of REST APIs that you can use to generate or validate proofs.
```
Comment (lucas-tortora) [inline documentation/docs/welcome.md]:
```suggestion
The extension exposes a custom set of REST APIs that you can use to generate or validate proofs of inclusion.
```

File: documentation/docs/reference/configuration.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+inx-poi -c config_defaults.json
+```
+
+To get the description of configuration parameters for the installed version of `inx-poi`, you can run the `inx-poi` command with `-h` and `--full` parameters:
+
+```bash
+inx-poi -h --full
+```
+
+## <a id="app"></a> 1. Application
```
Comment (lucas-tortora) [inline documentation/docs/reference/configuration.md]: The custom ids don't work with Docusarus.   I'm guessing they are there for the raw GitHub content, but they don't work either.  

I don't understand the numbers. They don't seem to add much to the content and would make direct # links a bit dirtier.

File: documentation/docs/reference/configuration.md (hunk 2 above)
Comment (aleksei-korolev) [reply documentation/docs/reference/configuration.md]: Good one.

File: documentation/docs/welcome.md (hunk 4 above)
Comment (aleksei-korolev) [reply documentation/docs/welcome.md:23]: I'd keep it as a node to keep it in line with the rest of the sections (configuration, API, source code)

**Review by lucas-tortora — APPROVED** [review]

**Comment by aleksei-korolev** [comment]
configuration.md is autogenerated, so I close this PR in favor of #24

**Pull request closed without merging**